src/decawave/src/decawave_node.cpp

```diff
@@ -50,54 +49,30 @@ void update_callback(const ros::TimerEvent&);
 
 // ROS Params
 double frequency = 50.0;
-int port_num = 0;
+std::string port_name= "ttyACM0";
 // for(int i = 0; i < argc; ++i){
 // int port_num = atoi(argv[1]);
 // }
-
 // Global_Vars
 Decawave *piTag;
-decawave_coordinate tagPos;
-// Decawave piTag(1);
-// decawave_coordinate tagPos;
 
 int main(int argc, char** argv){
-  // Init ROS
-  // ros::init(argc, argv, "decawave_node");
-  // nh = new ros::NodeHandle("");
-  // pnh = new ros::NodeHandle("~");
-
   // Params
```

Comment:
Ok, the order here is wonky. 
First init the ros node and allocate the node handles. 
Then get params (inputs):
```
    pnh->getParam("frequency",frequency);
```
We use the private node handle to access ros stuff private to the node, such as private params, while the global (yet namespaceable) node handle nh is to access global things like topics.